### PR TITLE
The location of the bootstrap iso image

### LINKED ISSUE #1351: Incorrect formulation of bootstrap iso path
The location of the bootstrap iso image should be set in the VCH config data:
https://github.com/vmware/vic/blob/master/lib/metadata/virtual_container_host.go#L111

and pulled from there when creating containers:
https://github.com/vmware/vic/blob/master/lib/portlayer/exec/handle.go#L221

### PR DESCRIPTION
**Pull request by caglar10ur**
should be set in the VCH config data.

Fixes #1351

### CODE REVIEW

**Comment by caglar10ur** [comment]
Pushed it by mistake, please ignore it for now

**Comment by fdawg4l** [comment]
I think this fixes #711 as well.  See https://github.com/vmware/vic/issues/711#issuecomment-226153713

File: lib/install/data/data.go [hunk 1]
```diff
@@ -68,6 +71,8 @@ type InstallerData struct {
 	KeyPEM  string
 	CertPEM string
 
+	BootstrapISO string
```
Comment (emlin) [inline lib/install/data/data.go]: You don't have to add value in this deprecated structure, cause you have new value in VirtualContainerHost spec.

File: lib/install/management/appliance.go [hunk 2]
```diff
@@ -223,6 +223,8 @@ func (d *Dispatcher) createApplianceSpec(conf *metadata.VirtualContainerHostConf
 	var devices object.VirtualDeviceList
 	var err error
 
+	conf.BootstrapImagePath = fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, conf.Name, vConf.BootstrapISO)
```
Comment (emlin) [inline lib/install/management/appliance.go]: Move this to validator, method basics(), and use input.BootstrapISO.

File: lib/install/validate/validator.go [hunk 3]
```diff
@@ -1185,6 +1185,8 @@ func (v *Validator) AddDeprecatedFields(ctx context.Context, conf *metadata.Virt
 	dconfig.ClusterPath = v.Session.Cluster.InventoryPath
 
 	dconfig.ResourcePoolPath = v.ResourcePoolPath
+	// This is the only way to move data from Data to InstallerData
+	dconfig.BootstrapISO = input.BootstrapISO
```
Comment (emlin) [inline lib/install/validate/validator.go]: Remove it.

File: lib/install/management/appliance.go (hunk 2 above)
Comment (caglar10ur) [inline lib/install/management/appliance.go]: done

File: lib/install/validate/validator.go (hunk 3 above)
Comment (caglar10ur) [inline lib/install/validate/validator.go]: done

**Comment by caglar10ur** [comment]
@emlin you could give this a another shot :)

**Comment by emlin** [comment]
LGTM 🐮

**Comment by dougm** [comment]
lgtm

**Comment by caglar10ur** [comment]
♻️

**Pull request merged**